osutil/nfs_linux.go

```diff
@@ -46,7 +46,7 @@ var isHomeUsingNFS = func() (bool, error) {
 		return false, fmt.Errorf("cannot parse %s: %s", etcFstab, err)
 	}
 	for _, entry := range fstab.Entries {
-		if (entry.Type == "nfs4" || entry.Type == "nfs") && (strings.HasPrefix(entry.Dir, "/home/") || entry.Dir == "/home") {
+		if (entry.Type == "nfs4" || entry.Type == "nfs" || entry.Type == "cifs") && (strings.HasPrefix(entry.Dir, "/home/") || entry.Dir == "/home") {
```

Comment:
I think it's time the modified `if`'s in this maybe better rewritten as a `switch entry.Type ...`